### ISSUE TITLE
[Random issue] Loss is above 6.9 of all 62000 iterations while training Inception V1

### ISSUE THREAD

**Issue by ZeweiChen11** [intel-analytics/BigDL#1185]
We run BigDL on 4-node cluster to train Inception V1 with batch size 704, learning rate 0.0896, default 62000 iterations . We meet the random issue for **2 times**. Loss is above 6.9 for all 62000 iteration.
Run cl:
`/opt/BigDL/dist/bin/bigdl.sh -- spark-submit --class com.intel.analytics.bigdl.models.inception.TrainInceptionV1 --master spark://bd01:7077 --driver-memory=30g --executor-cores=44 --num-executors=4 --total-executor-cores=176 --executor-memory=250g --driver-java-options="-Dlog4j.configuration=/opt/BigDL/log4j.conf" /opt/BigDL/dist/lib/bigdl-0.2.0-SNAPSHOT-jar-with-dependencies.jar --batchSize 704 -f hdfs:///user/root/sequence/ --checkpoint /mnt/disk2/inceptionV1_Spark2.1.1_standalone/ --learningRate 0.0896`

###################1st  time #####################################
2017-06-30 14:59:17 INFO  DistriOptimizer$:283 - [Epoch 1 0/1281167][Iteration 1][Wall Clock 0.0s] Train 704 in 182.19868339seconds. Throughput is 3.8639138 records/second. Loss is **6.907648**. Current learning rate is 0.0896. Current weight decay is 1.0E-4. Current momentum is 0.9.
…
2017-07-02 23:55:34 INFO  DistriOptimizer$:283 - [Epoch 35 81664/1281167][Iteration **61997**][Wall Clock 205155.258403651s] Train 704 in 3.317392829seconds. Throughput is 212.21484 records/second. Loss is **6.9102383**. Current learning rate is 7.196845186965593E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.
2017-07-02 23:55:37 INFO  DistriOptimizer$:283 - [Epoch 35 82368/1281167][Iteration **61998**][Wall Clock 205158.57579648s] Train 704 in 3.353730324seconds. Throughput is 209.9155 records/second. Loss is **6.904926**. Current learning rate is 6.232650759019548E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.
2017-07-02 23:55:41 INFO  DistriOptimizer$:283 - [Epoch 35 83072/1281167][Iteration **61999**][Wall Clock 205161.929526804s] Train 704 in 3.712935299seconds. Throughput is 189.6074 records/second. Loss is **6.906134**. Current learning rate is 5.088938034853138E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.
**2017-07-02 23:57:04 INFO  DistriOptimizer$:617 - Top1Accuracy is Accuracy(correct: 50, count: 50000, accuracy: 0.001)
2017-07-02 23:57:04 INFO  DistriOptimizer$:617 - Top5Accuracy is Accuracy(correct: 250, count: 50000, accuracy: 0.005)**
2017-07-02 23:57:09 INFO  DistriOptimizer$:283 - [Epoch 35 83776/1281167][Iteration **62000**][Wall Clock 205165.642462103s] Train 704 in 3.519127829seconds. Throughput is 200.04956 records/second. Loss is **6.905003**. Current learning rate is 3.598422593488989E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.

###################2nd  time #####################################
2017-07-06 10:25:06 INFO  DistriOptimizer$:283 - [Epoch 1 0/552059][Iteration 1][Wall Clock 0.0s] Train 704 in 22.186184978seconds. Throughput is 31.731457 records/second. Loss is **6.9078145**. Current learning rate is 0.0896. Current weight decay is 1.0E-4. Current momentum is 0.9.
…
2017-07-08 15:04:34 INFO  DistriOptimizer$:283 - [Epoch 79 539968/552059][Iteration **61998**][Wall Clock 189554.125527849s] Train 704 in 3.765112619seconds. Throughput is 186.9798 records/second. Loss is **6.9089756**. Current learning rate is 6.232650759019548E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.
2017-07-08 15:04:38 INFO  DistriOptimizer$:283 - [Epoch 79 540672/552059][Iteration **61999**][Wall Clock 189557.890640468s] Train 704 in 3.743832726seconds. Throughput is 188.04259 records/second. Loss is **6.9116993**. Current learning rate is 5.088938034853138E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.
**2017-07-08 15:05:09 INFO  DistriOptimizer$:617 - Top1Accuracy is Accuracy(correct: 15, count: 19316, accuracy: 7.765582936425761E-4)
2017-07-08 15:05:09 INFO  DistriOptimizer$:617 - Top5Accuracy is Accuracy(correct: 100, count: 19316, accuracy: 0.0051770552909505075)**
2017-07-08 15:05:14 INFO  DistriOptimizer$:283 - [Epoch 79 541376/552059][Iteration **62000**][Wall Clock 189561.634473194s] Train 704 in 3.802630935seconds. Throughput is 185.13498 records/second. Loss is **6.9057035**. Current learning rate is 3.598422593488989E-4. Current weight decay is 1.0E-4. Current momentum is 0.9.

**Comment by qiuxin2012** [comment]
Same issue with #1219, close this one.
